docs/datafeed.md

```diff
@@ -62,11 +62,10 @@ datafeed:
         maxIntervalMillis: 10000 # limit of the interval between two attempts
 ```
 
-The minimal configuration for the datafeed service is the version of the datafeed which will be chosen to be use in the
-BDK. For the moment, not all the customers have the datafeed version 2 available on their systems, that's why bot
-developers are able to choose the datafeed version that they wish to use on their bot. If the bot developers want to use
-the datafeed version 2 in their bot, the version configuration have to be specified as `v2`. Otherwise, the datafeed
-version 1 will be used by default.
+The minimal configuration for the datafeed service is the version to be used in the
+BDK. For the moment, not all customers have datafeed 2 available on their systems, that's why bot
+developers are able to choose the version that they wish to use on their bot. If the bot developers want to use
+datafeed 2, the version configuration have to be specified as `v2`. Otherwise, datafeed 1 will be used by default.
```

Comment:
I thought v2 had been set as default one, is not the case?

Reply:
Yes DFv2 is the default. I was just polishing the writing but the doc itself was outdated. Thanks for raising that, I corrected it in the last commit!